packages/utilities/psammead-styles/package.json

```diff
@@ -25,6 +25,6 @@
   ],
   "peerDependencies": {
     "react": "^16.13.1",
-    "@emotion/core": "^10.0.35"
+    "@emotion/react": "^11.1.4"
```

Comment:
Why was this changed? The `psammead-styles` package uses `@emotion/core` and `@emotion/styled`

Reply:
Hang on good point. Did I not change the imports within the component? Going to quickly put a DNM label on this

Reply:
Removed the import of @emotion/core for psammead-styles and replaced it with @emotion/react